### PR TITLE
Export joint limits

### PR DESCRIPTION
**Pull request by priority-kew**

### CODE REVIEW

**Comment by francesco-romano** [comment]
@priority-kew Do you mind adding a test checking for the limits in
https://github.com/priority-kew/idyntree/blob/export_joint_limits/bindings/pybind11/tests/test_idyntree_model.py#L127
and
https://github.com/priority-kew/idyntree/blob/export_joint_limits/bindings/pybind11/tests/test_idyntree_model.py#L151
?

Note that this might imply you also need to expose `hasPosLimits` (https://robotology.github.io/idyntree/master/classiDynTree_1_1IJoint.html#af4badbd5891621e847a61d767cffd25e) and `getPosLimits` (https://robotology.github.io/idyntree/master/classiDynTree_1_1IJoint.html#ab734574da90b6d2ae24169ea180a081b) in the bindings.

**Comment by traversaro** [comment]
The change seems nice, thanks, could you also update the CHANGELOG when you are done? In particular the support for exporting limits in the URDF ModelExporter could be useful also for user that do not use the pybind11 interface.

**Comment by priority-kew** [comment]
Ok, I've added tests to python and made a new minor version in the changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.3.0] - 2021-07-29
```
Comment (francesco-romano) [inline CHANGELOG.md:9]: I think you should add the changes to the "Unreleased" section and @traversaro will move it to a specific version when he will release it. But I will let him confirm on this

**Review by francesco-romano — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (traversaro) [reply CHANGELOG.md:9]: Yes, @francesco-romano  is correct, just deleting that header should be enough.
```suggestion
```

We are going to release quite soon, but I can't promise to do a release yesterday without breaking the rules of space and time. : ) 
See https://keepachangelog.com/en/1.0.0/#effort for more details on the idea of the `[Unreleased]` section.

**Review by traversaro — CHANGES_REQUESTED** [review]
See inline comment.

**Comment by priority-kew** [comment]
Gotcha, yeah I wasn't sure how releases worked.

**Review by traversaro — APPROVED** [review]

**Comment by traversaro** [comment]
Seems good to go to me, what do you think @francesco-romano ?

**Comment by francesco-romano** [comment]
LGTM

**Pull request merged**